MetaMask/metamask-extension
# feat: Connections List Item Network Badges

## **Description**
This PR adds Network badges to each item in the connection list. 
In future PRs, we'll make the onClick send to Nidhi's connect/disconnect flow
TODO: fix storybook


[![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/MetaMask/metamask-extension/pull/23397?quickstart=1)

## **Related issues**

Fixes: https://github.com/MetaMask/MetaMask-planning/issues/1978

## **Manual testing steps**

1. Open Extension
2. Make sure that Request Queueing Setting is turned on in Experimental tab
3. Connect to some Dapps
4. Go to All Permissions page and verify that the network for each connection is accurate.

## **Screenshots/Recordings**



### **Before**



### **After**


<img width="357" alt="image" src="https://github.com/MetaMask/metamask-extension/assets/10986371/c54a4b60-9998-4b06-9ece-60a65806c7b4">

## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
